hooks/use-fetch-exr.ts

```diff
@@ -26,42 +25,94 @@ export interface ExchangeRate {
   errorMessage: string | null;
 }
 
-export const useFetchExr = (date: string): ExchangeRate => {
-  const searchParams = useMemo(() => {
-    const p = new URLSearchParams({
-      format: "jsondata",
-      detail: "dataonly",
-    });
-    p.set("startPeriod", date);
-    p.set("endPeriod", date);
-    return p;
-  }, [date]);
+const fetchExchangeRate = async (date: string): Promise<number> => {
+  const searchParams = new URLSearchParams({
+    format: "jsondata",
+    detail: "dataonly",
+  });
+  searchParams.set("startPeriod", date);
+  searchParams.set("endPeriod", date);
+
+  return fetch(`${apiUrl}?${searchParams.toString()}`, { cache: "no-store" })
+    .then((res) => res.json())
+    .then(
+      (data: Response) =>
+        data.dataSets[0].series["0:0:0:0:0"].observations[0][0],
+    );
+};
 
-  const { data, isFetching, error } = useQuery({
+export const useFetchExr = (
```

Comment:
do we still need this if we have `useExchangeRates`?

Reply:
I was lazy, removing it right now :)